go/tasks/plugins/hive/client/qubole_client.go

```diff
@@ -62,7 +67,7 @@ type RequestBody struct {
 // Interface to interact with QuboleClient for hive tasks
 type QuboleClient interface {
 	ExecuteHiveCommand(ctx context.Context, commandStr string, timeoutVal uint32, clusterPrimaryLabel string,
-		accountKey string, tags []string) (*QuboleCommandDetails, error)
+		accountKey string, tags []string, commandMetadata CommandMetadata) (*QuboleCommandDetails, error)
```

Comment:
@EngHabu what's the impact of changing this interface wrt open source?  Do we just assume that no one is using it?  this would be a minor bump at least right? so we should add `#minor` to the PR name?  is that how it works?

Reply:
We discussed this offline. We will go with this change. Please update the title of the PR to start with `#minor` as Yee suggested.